recipes/busybox/busybox.inc

```diff
@@ -43,6 +43,7 @@ SRC_URI += "\
 "
 
 PACKAGES:<USE_busybox_udhcpc = "${PN}-udhcpc-scripts "
+SRC_URI:>USE_busybox_udhcpc = " file://udhcp-simple-script.patch"
```

Comment:
i don't think this falls under "sane defaults", if output like this is a problem it should be worked around downstream. Another possibility is to make it configurable, and not done by default.